### DISCUSSION TITLE
Allow Font to be configured.

### DISCUSSION THREAD

**Discussion by vHanda** [GitJournal/GitJournal#837] [Feature Requests]
Some users have requested that the Monospace font be the default. Perhaps we should just provide an option for them to choose a font?

**Comment by keelancook** [comment]
In addition to font choice, could font size potentially be configurable? The GitJournal Android app has the potential to be a real help to me. I use an Android phone and Chrome tablets and laptops. This one app works on all my devices. However, when I use it on anything larger than a tablet, the screen resolution makes the words very small. I'd love to be able to adjust font size in app, so that it's usable on larger screens.

**Comment by borjarobert** [comment]
I use 'nb' as my notes/todo app in my computer and the TODOs created that way always have a title that start with `[ ]`. Not that GitJournal becomes unusable at all (it works flawlessly, frankly), but it'd be a nice touch to use a monospace font so it looks better.

All the best!
